scripts/ccf/az-cleanroom-aci/up.sh

```diff
@@ -39,6 +39,10 @@ az-cleanroom-aci-up() {
     export KMS_SERVICE_CERT_PATH="$WORKSPACE/service_cert.pem"
     export KMS_MEMBER_CERT_PATH="$WORKSPACE/ccf-operator_cert.pem"
     export KMS_MEMBER_PRIVK_PATH="$WORKSPACE/ccf-operator_privk.pem"
+    export KMS_USER_CERT_PATH="$WORKSPACE/sandbox_common/user0_cert.pem"
+    export KMS_USER_PRIVK_PATH="$WORKSPACE/sandbox_common/user0_privk.pem"
+    export JWT_TOKEN_ISSUER_URL="http://localhost:3000/token"
```

Comment:
JWT Issuer details shouldn't be set here this is just about bringing up CCF. We already set them when bringing up the test issuer

Reply:
The comment https://github.com/microsoft/azure-privacy-sandbox-kms/pull/260#discussion_r1882471375 also applies here

Reply:
removed

Reply:
Great, my original comment about JWT issuer information still applies

Reply:
This isn't resolved